packages/lodestar/src/chain/chain.ts

```diff
@@ -7,13 +7,13 @@ import BN from "bn.js";
 import {EventEmitter} from "events";
 import {hashTreeRoot} from "@chainsafe/ssz";
 
-import {Attestation, BeaconBlock, BeaconState, number64, uint16, uint64} from "../types";
-import {DEPOSIT_CONTRACT_TREE_DEPTH, GENESIS_SLOT} from "../constants";
+import {Attestation, BeaconBlock, BeaconState, number64, uint16, uint64} from "@chainsafe/eth2.0-types";
+import {DEPOSIT_CONTRACT_TREE_DEPTH, GENESIS_SLOT} from "@chainsafe/eth2.0-constants";
```

Comment:
we need to add eslint plugin to force excplicit dependency declaration. Aka you are using dependency that you didn't declare here

Reply:
what do you mean?

Reply:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md

as we could use some package that we do not depend on but IDE let us import it. We can add it some later PR with rest of style thingies

Reply:
ohhhh gotcha :thumbsup:

Reply:
added eslint-plugin-import using your eslint configuration in the other eth2-types PR